PL/VR-PL-0000/rule.json

```diff
@@ -41,7 +41,8 @@
             "EU/1/20/1528",
             "EU/1/20/1507",
             "EU/1/21/1529",
-            "EU/1/20/1525"
+            "EU/1/20/1525",
+			"EU/1/21/1618"
```

Comment:
Did you consider also allowing the old coding for Nuvaxovid (NVX-CoV2373)? The vaccine is the same, it's just the coding which didn't exist before.

Reply:
We considered it, but decided to add new code only (at least for now)